mock_mpi/exec.py

```diff
@@ -62,7 +62,9 @@ def mock_mpiexec(nproc, target):
     ]
 
     # Make processes
-    procs = [Process(target=target, args=(comm,)) for comm in comms]
+    args = args or []
+    kwargs = kwargs or {}
+    procs = [Process(target=target, args=[comm] + args, kwargs=kwargs) for comm in comms]
```

Comment:
I think this will fail if args is a tuple rather than a list.  (Which I think should be allowed, since args are normally a tuple.)

So probably either `args=[comm] + list(args)` or `args=(comm,) + tuple(args)`.

Reply:
Done, and I added a test.